Formula/immudb.rb

```diff
@@ -25,6 +25,8 @@ def install
     ENV["WEBCONSOLE"] = "default"
     system "make", "all"
     bin.install %w[immudb immuclient immuadmin]
+
+    generate_completions_from_executable(bin/"immudb", "completion")
```

Comment:
can we also install completions for `immuclient` and `immuadmin`?

Reply:
sure!